enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v3_4/codegen/Methods.scala

```diff
@@ -60,15 +60,13 @@ object Methods {
   val endNode: MethodReference = method[RelationshipDataExtractor, Long]("endNode")
   val typeOf: MethodReference = method[RelationshipDataExtractor, Int]("type")
   val allConnectingRelationships: MethodReference = method[CompiledExpandUtils, RelationshipSelectionCursor]("connectingRelationships",
-                                                                                                             typeRef[ReadOperations],
                                                                                                              typeRef[Read],
                                                                                                              typeRef[CursorFactory],
                                                                                                              typeRef[NodeCursor],
                                                                                                              typeRef[Long],
                                                                                                              typeRef[Direction],
                                                                                                              typeRef[Long])
   val connectingRelationships: MethodReference = method[CompiledExpandUtils, RelationshipSelectionCursor]("connectingRelationships",
-                                                                                                          typeRef[ReadOperations],
```

Comment:
Wow, this is pretty big. Does that mean that the compiled runtime is fully ported?

Reply:
yes 🎆